src/PresentationalComponents/PathwaysPanel/PathwaysPanel.js

```diff
@@ -21,14 +21,12 @@ import { useSelector } from 'react-redux';
 
 const PathwaysPanel = () => {
   const intl = useIntl();
-  const { sort, offset } = useSelector(
-    ({ filters: { pathState } }) => pathState
-  );
+  const { offset } = useSelector(({ filters: { pathState } }) => pathState);
   const [expanded, setExpanded] = useState(
     JSON.parse(localStorage.getItem('advisor_pathwayspanel_expanded') || 'true')
   );
   const { data, isLoading, isFetching, isError } = useGetPathwaysQuery({
-    sort,
+    sort: '-recommendation_level',
```

Comment:
Wouldn't this break allowing users to sort by other columns? Like systems or name?

Reply:
The desired behavior is that pathway panels is static. We dont want it to be sorted any other way. Also as it is now, pathway panels reload every time pathway tables is sorted because the 'sort' filter in redux changes. @bastilian